### PR TITLE
Remove w2ui #3: Notebookbar

### PR DESCRIPTION
**Pull request by eszkadev**
- w2color conversion to JSDialog dropdown

### CODE REVIEW

File: browser/src/control/jsdialog/Widget.ColorPicker.ts [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+		colorPalette.push(JSON.parse(recentRow));
+	} else {
+		colorPalette.push(['F2F2F2', 'F2F2F2', 'F2F2F2', 'F2F2F2', 'F2F2F2', 'F2F2F2', 'F2F2F2', 'F2F2F2']); // recent colors (up to 8)
+	}
+
+	var palette = colorPalette;
+	if (typeof options == 'string') options = {
+		color: options,
+		transparent: true
+	};
```
[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/control/jsdialog/Widget.ColorPicker.ts]: done

File: browser/src/control/jsdialog/Widget.ColorPicker.ts [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+	customInput.placeholder = '#FFF000';
+	customInput.maxlength = 7;
+
+	for (let i = 0; i < customColors.length && i < 4; i++) {
+		createColor(customContainer, builder, customColors[i],
+				'8:0', undefined, data, options.color == customColors[i]);
+	}
+
+	const recentColors = palette[palette.length - 1];
+	const recentLabel = L.DomUtil.create('label', '', container)
```
[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/control/jsdialog/Widget.ColorPicker.ts]: done

File: browser/src/control/jsdialog/Widget.ColorPicker.ts [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+	const listbox = L.DomUtil.create('select', builder.options.cssClass + ' ui-listbox', paletteListbox);
+	for (const i in window.app.colorPalettes) {
+		const paletteOption = L.DomUtil.create('option', '', listbox);
+		if (i === currentPalette)
+			paletteOption.setAttribute('selected', 'selected');
+		paletteOption.innerText = window.app.colorPalettes[i].name;
+	}
+	L.DomUtil.create('span', builder.options.cssClass + ' ui-listbox-arrow', paletteListbox);
+
+	const detailedPalette = window.app.colorPalettes[currentPalette].colors
```
[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/control/jsdialog/Widget.ColorPicker.ts]: done

**Review by gokaysatir — APPROVED** [review]
Thanks.

**Pull request merged**